### PR TITLE
Add new todo item on pressing enter in between todo list rows

### PR DESCRIPTION
**Pull request by gkasoff**
Add new todo item on pressing enter in between todo list rows (https://github.com/upnotes-io/react-todo/issues/24#top)

### CODE REVIEW

File: package-lock.json
Comment (kamalkishor1991) [inline package-lock.json:1]: Please pull latest master. I have remove this file in master.

File: src/components/Todo/common/Todo/CompletedList/index.tsx [hunk 1]
```diff
@@ -39,6 +34,7 @@ export const TodoCompletedItem: FC<TodoCompletedItemProps> = ({
         <Checkbox
           checked
           onChange={() => {
+            console.log("completed list changed?");
```
Comment (kamalkishor1991) [inline src/components/Todo/common/Todo/CompletedList/index.tsx:37]: please remove

**Review by kamalkishor1991 — CHANGES_REQUESTED** [review]
@gkasoff thanks a lot for the pull request. 
sorry for the delay in reviewing this. I have added few minor comments. Please take a look.

File: src/components/Todo/common/Todo/CompletedList/index.tsx (hunk 1 above)
Comment (gkasoff) [reply src/components/Todo/common/Todo/CompletedList/index.tsx]: done

File: package-lock.json
Comment (gkasoff) [reply package-lock.json:1]: done

File: yarn.lock
Comment (kamalkishor1991) [inline yarn.lock:1]: please revert this. There should not be any changes in this file

File: package-lock.json
[the remark this answers is not in the document]
Comment (gkasoff) [reply package-lock.json:1]: sorry about that - file has been removed

**Comment by gkasoff** [comment]
@kamalkishor1991 hi is this all good now?

File: src/components/Todo/Todo.tsx [hunk 2]
```diff
@@ -5,6 +5,9 @@ import { Container } from "@material-ui/core";
 import { Item, TodoCompletedList } from "./common";
 import { Form } from "./common/Todo/Form";
 import { TodoItem } from "./common/types";
+import { isReturnStatement } from "@babel/types";
```
Comment (kamalkishor1991) [inline src/components/Todo/Todo.tsx]: please remove

**Review by kamalkishor1991 — APPROVED** [review]

**Pull request merged**